apps/pgsql/patroni/openshift/deployment.yaml

```diff
@@ -11,11 +11,11 @@ metadata:
     tags: postgresql
   name: patroni-pgsql-persistent
 labels:
-  app: ${INSTANCE}
+  app: ${NAME}${SUFFIX}
   phase: deploy
-  app.kubernetes.io/instance: ${INSTANCE}
+  app.kubernetes.io/instance: ${NAME}${SUFFIX}
   app.kubernetes.io/component: database
-  app.kubernetes.io/name: patroni
+  app.kubernetes.io/name: ${NAME}${SUFFIX}  
```

Comment:
I think `app.kubernetes.io/name` remains as `patroni` or we change to `postgresql` as per [kubernetes common labels](https://kubernetes.io/docs/concepts/overview/working-with-objects/common-labels/)

Reply:
Sure, we can leave this as something generic for reporting purposes, etc. You're right, patroni is likely best to help segregate. We can take the same approach to edb and generic postgres to help categorize them if needed.